Sources/Sentry/SentryClient.m

```diff
@@ -181,7 +181,15 @@ - (SentryEvent *)buildErrorEvent:(NSError *)error
 {
     SentryEvent *event = [[SentryEvent alloc] initWithError:error];
 
-    NSString *exceptionValue = [NSString stringWithFormat:@"Code: %ld", (long)error.code];
+    NSString *exceptionValue;
+
+    // If the error has a debug description, use that.
+    NSString *customExceptionValue = [[error userInfo] valueForKey:NSDebugDescriptionErrorKey];
+    if (customExceptionValue != nil) {
+        exceptionValue = customExceptionValue;
```

Comment:
`m`: I think we should include the error code in the exception value.
```
exceptionValue = [NSString stringWithFormat:@"Code: %ld - %@ custom", (long)error.code, customExceptionValue];
```

Reply:
My feeling is that if the user has opted to customise the message it should be displayed as is, the code is probably just going to add unnecessary noise. However, I guess adding “(code: xxx)” at the _end_ wouldn’t be too bad.